Vagrantfile

```diff
@@ -8,7 +8,7 @@ Vagrant.configure("2") do |config|
   # base box: Ubuntu 20.04 LTS (Focal Fossa)
   config.vm.box = "ubuntu/focal64"
 
-  vmhostname = `hostname`.strip+"-anyconnect" # derive the guest hostname from host's hostname
+  vmhostname = "vpn-anyconnect"
```

Comment:
@joscha-alisch: could you do this conditionally only for MacOS/Darwin?
see https://stackoverflow.com/a/39125251

Reply:
Sure, is there a benefit to taking the Hostname?